### DISCUSSION TITLE
从首页无法进入推荐的直播

### DISCUSSION THREAD

**Discussion by Yumeyo23** [ywmoyue/biliuwp-lite#543] [bug]
### Bug 描述

打开软件首页，推荐的视频是正常的，推荐的直播一旦点开就提示，出现一个未处理的错误，已记录，咱也不懂他记录在哪了，就提示这玩意，两台电脑，一台w10一台w11，都这样，

### 操作步骤

```plain text
1，打开软件首页
2，点击任意一个推荐的直播（不论是否关注）
3，成功出发
```

### 期望结果

正常播放直播（
上次反馈直播礼物栏，作者回复的好快，好有爱，爱了爱了

### 截图

![演示](https://github.com/ywmoyue/biliuwp-lite/assets/88522326/0afbea21-cbb5-4f8f-b8e3-42d685a5d171)
是GIF录制，有点儿大

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
前几天粗略看了一下，错误点不是出在卡片被点击后的链接解析上的，因为这个直播卡片本身携带的链接地址就为错误的“av1”。
由于时间不足没有继续往下挖。推测原因应该是b站变动了api返回的结构或者又是build值问题？
@ywmoyue 希望对问题的修复有帮助！

**Reply by ywmoyue** [reply]
@GD-Slime 感谢帮忙，我晚点也看一下
